vscode/webviews/components/modelSelectField/ModelSelectField.module.css

```
@@ -25,26 +28,30 @@
 
     & > svg {
         opacity: 0.5;
-        margin-top: -1px
     }
 }
 
+button > .model-title-with-icon .model-provider {
+    /* HACK(sqs): Hide the provider name from the button (show it only in the dropdown). */
+    display: none;
+}
+
 .badge  {
     margin-left: auto;
     line-height: 1;
-    letter-spacing: -0.025rem;
+    letter-spacing: -0.02rem;
+    font-size: 90%;
+    border-radius: 3px;
+    padding: 2px 3px 1px;
+    margin-bottom: -1px;
 }
 
 .cody-pro-badge {
-    background: repeating-conic-gradient(#7048E8 50%, #00CBEC 75%, #A112FF 100%, #FF5543 125%, #7048E8 150%);
-    color: rgba(255, 255, 255, 0.9);
-    border-radius: 4px;
-    padding: 3px 4px;
+    background: repeating-conic-gradient(#7048E888 50%, #00CBEC88 75%, #A112FF88 100%, #FF554388 125%, #7048E888 150%);
+    color: rgba(255, 255, 255, 1);
```

```suggestion
    background: linear-gradient(135deg, #60D5E7 14.14%, #CD8BFA 49.5%, #F4897D 70.71%);
    color: black;
```

Match the Figma gradient and make the text more readable on it